crates/sui-protocol-config/src/lib.rs

```diff
@@ -2094,6 +2114,10 @@ impl ProtocolConfig {
                 44 => {
                     // Enable consensus digest in consensus commit prologue on all networks..
                     cfg.feature_flags.include_consensus_digest_in_prologue = true;
+
+                    cfg.feature_flags.enable_soft_bundle = true;
```

Comment:
I think it's safer to release this to testnet-only first, before reaching mainnet.